dbflow/src/main/java/com/raizlabs/android/dbflow/single/FlowModelLoader.java

```diff
@@ -0,0 +1,22 @@
+package com.raizlabs.android.dbflow.single;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+
+import com.raizlabs.android.dbflow.config.FlowManager;
+import com.raizlabs.android.dbflow.sql.queriable.Queriable;
+import com.raizlabs.android.dbflow.structure.Model;
+
+/**
+ * Load a single model from the database.
+ *
+ * @param <TModel>
+ */
+@TargetApi(11)
+public class FlowModelLoader <TModel extends Model>
```

Comment:
#? How do you ensure the class is only parameterized by DBFlow `Model`-like classes? Is this no longer a concern with 4.x.x? Also, doing so will mean removing restriction on certain methods that should only be called with class that derive from `Model`, such as `registerForContentChanges`.